dev/packages/beats/osquery/0.0.1/dataset/result/elasticsearch/ingest-pipeline/default.json

```diff
@@ -14,14 +14,14 @@
         "ignore_failure": true
       }
     },
-    {<if .use_namespace >}
+    
     {
       "script": {
         "lang": "painless",
         "source": "def dict = ['result': new HashMap()]; for (entry in ctx['json'].entrySet()) { dict['result'][entry.getKey()] = entry.getValue(); } ctx['osquery'] = dict; ctx.remove('json');"
       }
     },
-    {< end >}
+    
```

Comment:
It is valid json with these empty lines?

Reply:
It passes the self-validation :)